### PR TITLE
More tests for edit case form and minor fixes identified from these tests

### PR DESCRIPTION
**Pull request by attwad**
- show "edit case" instead of "submit case" when editing a case
- pass initial value to chip input
- make fixture for full case pass yup validation

### CODE REVIEW

File: verification/curator-service/ui/cypress/fixtures/fullCase.json [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+        "additionalSources": [
+            {
+                "sourceUrl": "twitter.com/a-tweet"
+            },
+            {
+                "sourceUrl": "news.org/an-article"
+            }
+        ]
+    },
+    "sources": [
```
Comment (allysonjp715) [inline verification/curator-service/ui/cypress/fixtures/fullCase.json:16]: sources no longer exists in the case schema

File: verification/curator-service/ui/cypress/fixtures/fullCase.json [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        "values": [
+            "Hypertension",
+            "Type 2 diabetes",
+            "Coronary heart disease",
+            "Lung cancer"
+        ],
+        "status": "Yes"
+    },
+    "travelHistory": {
+        "numLocations": 1,
```
Comment (allysonjp715) [inline verification/curator-service/ui/cypress/fixtures/fullCase.json:109]: numLocations has also been removed

File: verification/curator-service/ui/cypress/integration/components/EditCase.spec.ts [hunk 3]
```diff
@@ -52,4 +52,33 @@ describe('Edit case', function () {
...
+            expect(resp.body.cases).to.have.lengthOf(1);
+            cy.visit(`/cases/edit/${resp.body.cases[0]._id}`);
+            // Check that we could parse the original case.
+            cy.contains('France');
+            cy.contains('Swedish');
+            cy.contains('Female');
+            cy.contains('Actor').should('not.exist');
+            // Change a few things.
+            cy.get('div[data-testid="sex"]').click();
+            cy.get('li[data-value="Female"').click();
```
Comment (allysonjp715) [inline verification/curator-service/ui/cypress/integration/components/EditCase.spec.ts:68]: It was Female before

Comment (attwad) [reply verification/curator-service/ui/cypress/integration/components/EditCase.spec.ts]: fixed

**Comment by attwad** [comment]
[Page to display a full case details](https://trello.com/c/OcIt0g8b/216-page-to-display-a-full-case-details)

**Pull request merged**